### PR TITLE
Issue #5484: Move Clear log messages button to bottom of page.

### LINKED ISSUE #5484: Move the 'Clear log messages' button to the bottom of the page
This is a follow-on from https://github.com/backdrop/backdrop-issues/issues/2353. There, after much discussion, it was finally decided that the 'Clear log messages' button at the top of the 'Recent log messages' (DBLog) page will be moved to the bottom of the page instead.

The reasons for this will not be repeated here (see the other issue for all the details about that). This issue will focus on getting a PR ready to test and implement this change.

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/5484

### CODE REVIEW

File: core/modules/dblog/dblog.admin.inc [hunk 1]
```diff
@@ -19,18 +19,17 @@ function dblog_overview() {
...
-    WATCHDOG_EMERGENCY     => 'dblog-emergency',
+    WATCHDOG_DEBUG => 'dblog-debug',
+    WATCHDOG_INFO => 'dblog-info',
+    WATCHDOG_NOTICE => 'dblog-notice',
+    WATCHDOG_WARNING => 'dblog-warning',
+    WATCHDOG_ERROR => 'dblog-error',
+    WATCHDOG_CRITICAL => 'dblog-critical',
+    WATCHDOG_ALERT => 'dblog-alert',
+    WATCHDOG_EMERGENCY => 'dblog-emergency',
     WATCHDOG_DEPRECATED => 'dblog-deprecated',
```
Comment (klonos) [inline core/modules/dblog/dblog.admin.inc:30]: 👍🏼

File: core/modules/dblog/dblog.admin.inc [hunk 2]
```diff
@@ -19,18 +19,17 @@ function dblog_overview() {
...
-    WATCHDOG_ALERT     => 'dblog-alert',
-    WATCHDOG_EMERGENCY     => 'dblog-emergency',
+    WATCHDOG_DEBUG => 'dblog-debug',
+    WATCHDOG_INFO => 'dblog-info',
+    WATCHDOG_NOTICE => 'dblog-notice',
+    WATCHDOG_WARNING => 'dblog-warning',
+    WATCHDOG_ERROR => 'dblog-error',
+    WATCHDOG_CRITICAL => 'dblog-critical',
+    WATCHDOG_ALERT => 'dblog-alert',
+    WATCHDOG_EMERGENCY => 'dblog-emergency',
```
Comment (klonos) [inline core/modules/dblog/dblog.admin.inc:29]: Nice 👍🏼

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by quicksketch** [comment]
Merged into 1.x and 1.21.x. Thanks folks!

**Pull request merged**